src/toxext.h

```diff
@@ -97,24 +110,31 @@ struct ToxExtPacketList *toxext_packet_list_create(struct ToxExt *toxext,
  * Appends an extension message onto an existing packet. This is the endpoint
  * extensions should use when trying to send their own data.
  */
+ // "existing packet" -> "existing packet list"?
+ // "This is the endpoint extensions" -> "This is the endpoint the extensions"
+ // not sure about "endpoint" in general vs API or function :shrug:
 int toxext_packet_append(struct ToxExtPacketList *packet /*in/out*/,
 			 struct ToxExtExtension *extension, void const *data,
 			 size_t size);
 
 /**
  * Sends and frees an existing packet
  */
+ // "packet" -> "packet list"? was called a "packet list" in toxext_packet_list_create, and that seems more accurate
 int toxext_send(struct ToxExtPacketList *packet);
 
 /**
  * Determines if the input data is meant for ToxExt
  */
+ // maybe some clarification on "input data", I'm guessing this means tox custom packet data
 bool is_toxext_packet(uint8_t const *data, size_t size);
 
 /**
  * Callback for lossless custom packets from toxcore. Clients are expected to
  * attach this to their toxcore callbacks
  */
+ // isn't this not really a callback? maybe more like a "Handling function for lossless custom packets from toxcore"?
```

Comment:
I think I was thinking that they call it from the toxcore callback. Maybe calling it a handler makes more sense?